projects/go-lib/src/lib/components/go-toast/go-toast.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="go-toast">
+  <div class="go-toast-strip" [ngClass]="statusClasses()"></div>
+  <div class="go-toast-status">
+    <go-icon class="go-toast-status__icon" [ngClass]="statusClasses()" icon="{{ goIconType() }}"></go-icon>
+  </div>
+  <div class="go-toast-content"  [ngClass]="{ 'go-toast-content--no-title': !header }">
+    <h5 class="go-toast-content__title" *ngIf="header">{{ header }}</h5>
+    <p class="go-toast-content__message">{{ message }}</p>
+  </div>
+  <div class="go-toast-dismiss" *ngIf="dismissable">
+    <go-icon class="go-toast-dismiss__icon" icon="close" (click)="dismiss()"></go-icon>
```

Comment:
This should be wrapped in a button so we know that it is clickable. We will also get access to this action via that keyboard.